subjects/java/checkpoints/is-anagram/README.md

```diff
@@ -0,0 +1,45 @@
+## Is Anagram
+
+### Instructions
+
+Create a class `AnagramChecker` that provides a method to check if two strings are anagrams. An anagram is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.
```

Comment:
Is the check going to be case-sensitive?
It might be worth saying that two empty strings should be considered anagrams of each other

Reply:
no need to specify the case of empty string because it's logically correct if think of it, then they need to know it by them selves.